gossip/ethapi_backend.go

```diff
@@ -168,12 +174,12 @@ func (b *EthAPIBackend) GetConsensusTime(ctx context.Context, shortEventID strin
 }
 
 // GetHeads returns IDs of all the epoch events with no descendants.
-func (b *EthAPIBackend) GetHeads(ctx context.Context, epoch int) (heads hash.Events, err error) {
+func (b *EthAPIBackend) GetHeads(ctx context.Context, epoch rpc.BlockNumber) (heads hash.Events, err error) {
 	current := b.svc.engine.GetEpoch()
 
 	var requested idx.Epoch
 	switch {
-	case epoch == -1:
+	case epoch == rpc.LatestBlockNumber:
```

Comment:
how to query getHeads() of the current epoch in one shot?
can you include the case 
epoch==-1 to return the current epoch?

Any change to the logic of an API method must include an update the comment of the method as well.

Reply:
Pass "latest" to get heads for latest epoch, like that: `{"jsonrpc":"2.0","method":"debug_getHeads","params":["latest"],"id":1}`
I.e. it's the same as for web3 methods, like getBlockByNumber